internal/rpc/msg/as_read.go

```diff
@@ -147,7 +148,12 @@ func (m *msgServer) MarkConversationAsRead(
 		for i := hasReadSeq + 1; i <= req.HasReadSeq; i++ {
 			seqs = append(seqs, i)
```

Comment:
simplify code

```go
		for i := hasReadSeq + 1; i <= req.HasReadSeq; i++ {
			//avoid client missed call MarkConversationMessageAsRead by order
			if !utils2.Contain(val, seqs...) {
				seqs = append(seqs, val)
			}
		}

```

Reply:
Dear [@rfyiamcoo](https://github.com/rfyiamcool),

Thank you for taking the time to review the code. I sincerely appreciate your input and suggestions. 

Regarding the comment you made about simplifying the code, I have carefully considered your feedback and made the necessary changes. 

```go
for i := hasReadSeq + 1; i <= req.HasReadSeq; i++ {
    // Avoid client missed call MarkConversationMessageAsRead by order
    if !utils2.Contain(val, seqs...) {
        seqs = append(seqs, val)
    }
}
```

I believe these modifications will enhance the readability and maintainability of the code. Your expertise and attention to detail have been invaluable in improving the quality of the codebase. 

Once again, thank you for your review. Your feedback has been instrumental in refining the code and making it more efficient. If you have any further suggestions or comments, please do not hesitate to let me know. 

I genuinely appreciate your time and effort in helping me improve the code.